### ISSUE TITLE
A New Start

### ISSUE THREAD

**Issue by JiveyGuy** [Open-Game-Cafe-Foundation/open_cyber_center#1]
Okay let's make plans about this software, we are starting from nothing! 

Let's come up with 3 main goals for this project as a whole:

Feel free to change or edit them, or come up with new ones altogether. 

- Be a gaming-cafe software with a focus on being a **real feasible option** for game centers. 
- Being simple to set up and manage
- Having a small process footprint

**Comment by courtmuel** [comment]
I think the goals you have listed are good and are all achievable especially with our prior knowledge! Also side note, you can have more than one owner for the Organization - if you would like me to change your permissions to add you as one please let me know and I can do that for you.

**Comment by JiveyGuy** [comment]
Awesome, glad you like the goals!

Let's make a plan to realize these goals. 

I am okay with just being a contributor, maybe there will be a good reason for me to have more power but for now, this is fine.

###  PLAN

1. Decide on specific requirements of OCC (Open Cyber Center) 
2. Make a high-level design of OCC components
3. Decide on tools, packages, and services for the project. 
4. Make a plan for development milestones and feature goals

Does this sound like a good plan?

**Comment by courtmuel** [comment]
Yes this all sounds good to me. Would you like to use Trello to keep track of stuff again I can set that up for us if need be. Or do you have any idea on how to decide on the flow of progress? Perhaps a flowchart or something of the sort.

**Comment by JiveyGuy** [comment]
> Yes this all sounds good to me. Would you like to use Trello to keep track of stuff again I can set that up for us if need be. Or do you have any idea on how to decide on the flow of progress? Perhaps a flowchart or something of the sort.

I think using GitHub issues to track tasks is a good idea, it's how many projects do it.

**Comment by JiveyGuy** [comment]
Going to open a new issue for step 1.

**Comment by courtmuel** [comment]
Sounds great I will be keeping my eye out and will attempt to get started as soon as I can.

**Comment by JiveyGuy** [comment]
Just posted it, also, impressive response time!

**Comment by JiveyGuy** [comment]
Incase you have notifs off for other threads, I responded in the other issue.
